src/code/z_scene.c

```diff
@@ -57,21 +186,41 @@ GLOBAL_ASM("./asm/nonmatching/z_scene/Scene_HeaderCommand05.asm")
 
 GLOBAL_ASM("./asm/nonmatching/z_scene/Scene_HeaderCommand13.asm")
 
-GLOBAL_ASM("./asm/nonmatching/z_scene/Scene_HeaderCommand09.asm")
+// func_801303D0
+// Scene Command 0x09: Undefined
+void Scene_HeaderCommand09(GlobalContext* ctxt, SceneCmd* entry) {
+	GlobalContext* a0 = ctxt;
```

Comment:
These assignments are probably unneeded, since the arguments will already be in a0 and a1. Try just `{}` and see if it matches.